### ISSUE TITLE
Warning when using role="text" on heading elements

### ISSUE THREAD

**Issue by jackdomleo7** [jackdomleo7/Checka11y.css#107]
### Describe the new a11y feature or project enhancement

The new `role="text"` is nice because it prevents a screenreader from breaking up sentences with `<span>` or `<strong>` inside, however when using it on a heading, it causes the heading to lose its semantic meaning.

### Describe the solution you'd like

```scss
h1, h2, h3, h4, h5, h6 {
  &[role="text"] {
    // Show warning
  }
}
```

### Link(s)

https://openinclusion.com/blog/using-role-text-enhance-mobile-screen-reader-output/

**Comment by emmalearnscode** [comment]
Can I work on this one?

**Comment by jackdomleo7** [comment]
@emmalearnscode absolutely!

**Comment by emmalearnscode** [comment]
Hi Jack, when I run npm install it fails and tells me I need to install Python - is this something I need to do or is something else going wrong? I can't see any python files in the code and it's not mentioned in the contributing instructions.

**Comment by jackdomleo7** [comment]
Hi @emmalearnscode , correct, there is no Python in this project. But there is Sass which is built using Python I believe in the background. Maybe this StackOverflow helps? I encountered this when I first installed Sass and can't remember the exact solution https://stackoverflow.com/questions/45801457/node-js-python-not-found-exception-due-to-node-sass-and-node-gyp/45807410#:~:text=The%20error%20message%20means%20that,hasn't%20been%20installed%20yet.

**Comment by emmalearnscode** [comment]
Ok, thanks! I'll see if I can get it up and running tomorrow. I've already added most of the changes, just need to get it to build and run tests.
